ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml

```diff
@@ -73,8 +73,6 @@
     <option name="ALIGN_MULTILINE_PARAMETERS_IN_CALLS" value="true" />
     <option name="ALIGN_MULTILINE_FOR" value="false" />
     <option name="ALIGN_MULTILINE_EXTENDS_LIST" value="true" />
-    <option name="CALL_PARAMETERS_WRAP" value="2" />
-    <option name="METHOD_PARAMETERS_WRAP" value="2" />
```

Comment:
@hasys Can you check if eclipse rules are currently in sync with this?

Reply:
It is not 100% similar, because it is not possible, but it is similar now. Main points are:
* Eclipse won't override alignment from IDEA in this place
* you can man manually wrap parameters on your choose and Eclipse won't change it back
* parameters will be aligned in a column if new line present

Reply:
Thanks Kirill, that actually sounds like 100% IDEA's behavior. Time will tell...